Makefile

```diff
@@ -0,0 +1,7 @@
+make create-db:
+	docker run --name admin -p "5432:5432" -e POSTGRES_DB=admin -e POSTGRES_USER=admin -e POSTGRES_PASSWORD=admin -d "postgres:14-bullseye"
+	until docker exec admin pg_isready -U admin
```

Comment:
[CHANGE REQUEST] I am unsure that it needs.

Reply:
[SUGGESTION] It would be better if you used `docker compose`.

Reply:
I use it only for local developing. It useful for fast run a code